moped-toolbox/knack_migration/1_prepare_projects.py

```diff
@@ -402,7 +402,6 @@ def set_sponsor_and_partners(projects, entities_index):
         if partner:
             entity = entities_index[partner]
             new_partner = {}
-            new_partner["partner_name"] = entity["entity_name"]
```

Comment:
figured this could go too - @chiaberry

Reply:
will we ever use these toolbox scripts again to import projects from knack @johnclary? If we might someday, then yes, move this. But if we won't, then I guess its more like a historical record? and then it doesnt matter to me if it stays or goes.

Reply:
i very much hope we never need to run these scripts again, so let's leave those old toolbox script frozen in time. i'll pull this commit outta here so we can get this merged.